src/main/java/uk/org/okapibarcode/output/PostScriptRenderer.java

```diff
@@ -61,10 +64,24 @@ public class PostScriptRenderer implements SymbolRenderer {
      * @param ink the ink (foreground) color
      */
     public PostScriptRenderer(OutputStream out, double magnification, Color paper, Color ink) {
+        this(out, magnification, paper, ink, 0);
+    }
+
+    /**
+     * Creates a new PostScript renderer with the specified clockwise rotation.
```

Comment:
No need to mention the rotation in the method description, can be same as other constructor.